src/utils/ValidationHelper.ts

```diff
@@ -15,9 +15,7 @@ export class ValidationHelper {
 			await validateOrReject(model, { forbidUnknownValues: true });
 		} catch (errors) {
 			const errorDetails = this.getErrors(errors);
-			console.log(`${model.constructor.name}`);
-			console.log("**** Error validating " + `${model.constructor.name}` + "   " + JSON.stringify(errorDetails));
-			console.log(`Failed to validate data - ${model.constructor.name}`, "ValidationHelper", HttpCodesEnum.UNPROCESSABLE_ENTITY, errorDetails);
+			logger.error({ message: `ValidationHelper: Failed to validate data ${model.constructor.name}` + JSON.stringify(errorDetails) });
```

Comment:
`errorDetails` contains something called `value` - what is that? Want to make sure it couldn't be PII

Reply:
I guess it could populate the value incase it doesnt match the constraint. Most of the fields in a Modal are defined with @IsString(),@IsNotEmpty() or @IsArray() in which case the value will not be populated if its empty or may log the non-string non-array values. Even email constraint may display the invalid email value in logs. Would be safer if we remove the value from the getErrors function to avoid any PII being logged.

Reply:
Cool makes sense thanks for doing that! Could we please also add a messageCode to this and update the runbook so that it's consistent with our other error logs?

Reply:
This is a utility function being called commonly. We are already capturing the error codes at the caller function, do we need it here as well? I have to use a code which would be generic like MODAL_VALIDATION_FAILED

Reply:
I see what you mean. I would add it for consistency but sounds like it's not 100% needed to happy to leave the decision to you 👍